CommonTools/RecoAlgos/plugins/MuonRefPatSelector.cc

```diff
@@ -0,0 +1,13 @@
+#include "FWCore/Framework/interface/MakerMacros.h"
+#include "CommonTools/UtilAlgos/interface/SingleObjectSelector.h"
+#include "CommonTools/UtilAlgos/interface/StringCutObjectSelector.h"
+#include "DataFormats/PatCandidates/interface/Muon.h"
+
+
+ typedef SingleObjectSelector<
+           pat::MuonCollection, 
+           StringCutObjectSelector<pat::Muon>,
+           pat::MuonRefVector
+         > MuonRefPatSelector;
```

Comment:
PhysicsTools/PatAlgos/plugins/PATObjectSelector.h seems like a more appropriate place to define this.